python-backend/app/api/mines/variances/resources/variance.py

```diff
@@ -104,21 +85,18 @@ class VarianceResource(Resource, UserMixin, ErrorMixin):
             'variance_id': 'ID of the variance to fetch'
         })
     @requires_any_of([MINE_VIEW])
-    @api.marshal_with(variance_model, code=200)
+    @api.marshal_with(VARIANCE_MODEL, code=200)
     def get(self, mine_guid, variance_id):
         variance = Variance.find_by_variance_id(variance_id)
 
         if variance is None:
-            raise BadRequest('Unable to fetch variance')
+            raise NotFound('Unable to fetch variance')
 
         return variance
 
 
 class VarianceDocumentUploadResource(Resource, UserMixin, ErrorMixin):
-    parser = reqparse.RequestParser(trim=True)
-    parser.add_argument('mine_document_guid', type=str)
-    parser.add_argument('document_manager_guid', type=str)
-    parser.add_argument('document_name', type=str)
+    parser = CustomReqparser()
```

Comment:
Is this used? 

the post parses arguments (but there are none). 
do we like the parser arguments being added in the function? I feel like that's going to get awkward and harder to find things. I'd rather see the resources parser have all it's arguments defined in one spot.

Reply:
Well, I can define them all here and then redefine/override them in the methods that have different required args?